runtests.py

```diff
@@ -28,9 +28,11 @@
     import platform
     IS_PYPY = platform.python_implementation() == 'PyPy'
     IS_CPYTHON = platform.python_implementation() == 'CPython'
+    CAN_SYMLINK = platform.system() != 'Windows'
 except (ImportError, AttributeError):
     IS_CPYTHON = True
     IS_PYPY = False
+    CAN_SYMLINK = False
 IS_PY2 = sys.version_info[0] < 3
```

Comment:
I don't think we need to `platform` module for this.
```suggestion
    import platform
    IS_PYPY = platform.python_implementation() == 'PyPy'
    IS_CPYTHON = platform.python_implementation() == 'CPython'
except (ImportError, AttributeError):
    IS_CPYTHON = True
    IS_PYPY = False

IS_PY2 = sys.version_info[0] < 3
CAN_SYMLINK = sys.platform != 'win32'
```